### PR TITLE
chore: NonAncientEventWindow to EventWindow

### LINKED ISSUE #12607: Rename NonAncientEventWindow to EventWindow
A while back we agreed to rename `NonAncientEventWindow` to `EventWindow`, but delayed making the change due to in-flight PRs. Now it should be safe to make that change.

### PR DESCRIPTION
**Pull request by cody-littley**
closes #12607

### CODE REVIEW

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/creation/tipset/TipsetEventCreator.java [hunk 1]
```diff
@@ -483,7 +483,7 @@ public void clear() {
     public String toString() {
         final StringBuilder sb = new StringBuilder();
         sb.append("Minimum generation non-ancient: ")
```
Comment (alittley) [inline platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/creation/tipset/TipsetEventCreator.java:485]: Now might be a good time to update this out-of-date field label

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/InOrderLinkerWiring.java [hunk 2]
```diff
@@ -30,14 +30,14 @@
  * Wiring for the {@link InOrderLinker}.
  *
  * @param eventInput                 the input wire for events to be linked
- * @param nonAncientEventWindowInput the input wire for the minimum generation non-ancient
+ * @param eventWindowInput the input wire for the minimum generation non-ancient
```
Comment (alittley) [inline platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/InOrderLinkerWiring.java:33]: doc is out of date (and misaligned)

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/components/PcesWriterWiring.java [hunk 3]
```diff
@@ -33,7 +33,7 @@
  *                                                 complete
  * @param eventInputWire                           the input wire for events to be written
  * @param discontinuityInputWire                   the input wire for PCES discontinuities
- * @param nonAncientEventWindowInput               the input wire for non ancient event windows
+ * @param eventWindowInput               the input wire for non ancient event windows
```
Comment (alittley) [inline platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/components/PcesWriterWiring.java:36]: doc mentions "non ancient event windows" (and is misaligned)

File: platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/event/orphan/OrphanBufferTests.java [hunk 4]
```diff
@@ -206,34 +206,34 @@ private GossipEvent createRandomEvent(
     /**
      * Check if an event has been emitted or is ancient
      *
-     * @param event                 the event to check
-     * @param nonAncientEventWindow the non-ancient event window defining ancient.
+     * @param event       the event to check
+     * @param eventWindow the  event window defining ancient.
```
Comment (alittley) [inline platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/event/orphan/OrphanBufferTests.java:210]: unnecessary extra space between "the" and "event"

File: platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/event/orphan/OrphanBufferTests.java [hunk 5]
```diff
@@ -206,34 +206,34 @@ private GossipEvent createRandomEvent(
...
 
     /**
      * Assert that an event should have been emitted by the orphan buffer, based on its parents being either emitted or
      * ancient.
      *
-     * @param event                 the event to check
-     * @param nonAncientEventWindow the non-ancient event window defining ancient.
-     * @param emittedEvents         the events that have been emitted so far
+     * @param event         the event to check
+     * @param eventWindow   the non-ancient event window defining ancient.
```
Comment (alittley) [inline platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/event/orphan/OrphanBufferTests.java:226]: Doc still mentions "non-ancient event window"

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/EventWindow.java [hunk 6]
```diff
@@ -28,10 +28,9 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 
 /**
- * Determines the non-ancient lower bound (inclusive) on events and communicates the window of rounds between the
- * pendingConsensusRound and the minimumRoundNonAncient (inclusive).
+ * Describes the current window of events that the platform is using.
  */
-public class NonAncientEventWindow {
+public class EventWindow {
```
Comment (alittley) [inline platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/EventWindow.java:33]: Searching for regex `non.?ancient.?event.?window` yields a couple missed instances of the old terminology

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/creation/tipset/TipsetEventCreator.java (hunk 1 above)
Comment (cody-littley) [reply platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/creation/tipset/TipsetEventCreator.java:485]: fixed

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/InOrderLinkerWiring.java (hunk 2 above)
Comment (cody-littley) [reply platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/InOrderLinkerWiring.java:33]: fixed

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/components/PcesWriterWiring.java (hunk 3 above)
Comment (cody-littley) [reply platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/components/PcesWriterWiring.java:36]: fixed

File: platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/event/orphan/OrphanBufferTests.java (hunk 4 above)
Comment (cody-littley) [reply platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/event/orphan/OrphanBufferTests.java:210]: fixed

File: platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/event/orphan/OrphanBufferTests.java (hunk 5 above)
Comment (cody-littley) [reply platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/event/orphan/OrphanBufferTests.java:226]: fixed

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/EventWindow.java (hunk 6 above)
Comment (cody-littley) [reply platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/EventWindow.java:33]: I only found one (although there may have been a few I fixed based on your previous comments). I fixed the one that I did find.

**Review by alittley — APPROVED** [review]

**Review by alittley — DISMISSED** [review]

**Review by edward-swirldslabs — DISMISSED** [review]

**Review by alittley — APPROVED** [review]

**Review by edward-swirldslabs — APPROVED** [review]

**Pull request merged**